packages/rest/src/__tests__/unit/router/routing-table.unit.ts

```diff
@@ -214,6 +214,44 @@ function runTestsWithRouter(router: RestRouter) {
     expect(route.pathParams).to.containEql({userId: '1', format: 'json'});
   });
 
+  it('finds "GET /orders/ and GET /orders/{id}" endpoints', () => {
+    class TestController {
+      @get('/orders/{id}')
+      async getOrderById(@param.path.number('id') id: number): Promise<object> {
+        return {id};
+      }
+      @get('/orders')
+      async findOrders(): Promise<object[]> {
+        return [];
+      }
+      // A path overlaps with `/order/{id}`. Please note a different var name
```

Comment:
```suggestion
      // A path overlaps with `/orders/{id}`. Please note a different var name
```

Reply:
nitpick: `A path overlaps` -> `A path that overlaps`